### PR TITLE
DRTM from iPXE tests for Asrock r1000v platform

### PR DESCRIPTION
**Pull request by pkonkol**

### CODE REVIEW

File: keywords.robot [hunk 1]
```diff
@@ -82,23 +104,23 @@ iPXE boot entry
...
 
 GRUB boot entry
     [Documentation]    Enter specified in argument iPXE menu entry.
-    [Arguments]    ${menu_entry}
-    Set Timeout    30s
-    ${move}=    GRUB get menu position    ${menu_entry}
-    : FOR    ${INDEX}    IN RANGE    0    ${move}
+    [Arguments]    ${menu_entry}    ${reference_str}    ${rs_offset}
+    ${move}=    GRUB get menu position    ${menu_entry}    ${reference_str}    ${rs_offset}
+    ${grub_key}=    Set Variable If    ${move} < 0    ${grub_key_up}    ${grub_key}
```
Comment (lwcislo) [inline keywords.robot:126]: @pkonkol Please add some documentation on what is happening here.

File: keywords.robot [hunk 2]
```diff
@@ -343,13 +349,23 @@ Prepare Test Suite
...
+    ...        ${CURDIR}/platform-configs/supermicro-m11sdv-8ct-ln4f.robot
+    ...        AND    Set Library Search Order    supermicro-m11sdv-8ct-ln4f
+    Run Keyword If   '${dev_type}' not in ['auto', 'None']
+    ...               Set Storage Device Number And Type
 
     Open Connection And Log In
     ${platform}=    Get current RTE param    platform
     Set Global Variable    ${platform}
     Get DUT To Start State
+    Telnet.Set Timeout    90s
```
Comment (lwcislo) [inline keywords.robot]: @pkonkol It should be moved to test Settings, it is used in each test.

**Review by lwcislo — CHANGES_REQUESTED** [review]

File: keywords.robot (hunk 1 above)
Comment (pkonkol) [reply keywords.robot:126]: Added

File: keywords.robot (hunk 2 above)
Comment (pkonkol) [reply keywords.robot]: Moved

**Review by lwcislo — APPROVED** [review]

**Pull request merged**